index.js

```diff
@@ -68,9 +68,10 @@ module.exports = function(version, _options) {
 
             return config.join('');
         },
-        compile: function(language, step, options) {
-            if (!language) throw new Error('No language code provided');
-            if (languages.supportedCodes.indexOf(language) === -1) throw new Error('language code ' + language + ' not loaded');
+        compile: function(originalLanguage, step, options) {
+            if (!originalLanguage) throw new Error('No language code provided');
+            var language = this.findAppropriateLanguage(originalLanguage);
```

Comment:
Several other exported methods take a language parameter as well. We should be consistent; otherwise, a client using the `tokenize()` method, for example, will end up with a mismash.